docs/serving/services/creating-services.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Creating Knative Services
+type: “docs”
+---
+
+Knative services are used to deploy an application. Knative Services attempt to model a more logical definition of a Service or Application. To achieve this, Services use the concept of Revisions which are each backed by a deployment with 2 Kubernetes Services.   
```

Comment:
Capitalize "service"

Also, I'd rewrite this to refer to a Knative Service, singular, since there's usually just one Knative Service.